### ISSUE TITLE
Enquiry about max_eval_steps in the released example

### ISSUE THREAD

**Issue by xiaobo-xing** [snap-stanford/relational-transformer#4]
Hi, thanks for releasing the code for this insightful work. In `scripts/example_finetune.py`, `max_eval_steps` is set to 40 for validation. For the supervised finetuning results reported in the paper, did you evaluate on the full validation set (e.g., max_eval_steps=-1), or also with a capped `max_eval_steps = 40`? Thanks!

**Comment by rishabh-ranjan** [comment]
Hi, for efficiency reasons we used `max_eval_steps = 40`. Under our setup, this subsamples 10k random examples from the test set. We have found that this gives a reasonable estimate of the test metrics (within 2-3 decimal places) as compared to `max_eval_steps = -1`.

**Comment by xiaobo-xing** [comment]
Thank you!
